deploy/README.md

```diff
@@ -0,0 +1,8 @@
+# Deploying the search client to a custom Wordpress page template
+
+1. Run `npm run build` to build this project.
+2. Figure out which Wordpress theme you're using and navigate to the directory for that theme under wp\_content.
```

Comment:
Is `wp_content` a directory in Wordpress?

Reply:
it should be wp-content, but yes it is.